### PR TITLE
New module: RollbarSwift

### PR DESCRIPTION
**Pull request by akornich**
## Description of the change

> Added new RollbarSwift module. It helps in handling NSExceptions within Swift code.
> Added macosAppSwift demo app demonstrating use of the RollbarSwift module
> Prepared for v2.0.0-beta3

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> n/a

### Development

- [x] Lint rules pass locally
- [x] The code changed/added as part of this pull request has been covered with tests
- [x] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: RollbarSwift/Sources/RollbarSwift/RollbarExceptionGuard.m [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    
+    [RollbarTryCatch try:^(void) {
+            
+            tryBlock();
+            success = YES;
+        }
+                       catch:^(NSException *exception) {
+            
+            exceptionError = [self convertException:exception];
+
```
Comment (bishopb) [inline RollbarSwift/Sources/RollbarSwift/RollbarExceptionGuard.m:35]: Nit: What's up with the indentation here? Is this a generated file, or can we fix this up?

File: Demos/macosAppSwift/macosAppSwift/ContentView.swift [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        let exceptionGuard = createGuard();
+        
+        var success = true;
+        
+        success = exceptionGuard.tryExecute {
+            
+            self.generateObjCException();
+        }
+
+        print("Guearded execution succeeded: \(success).")
```
[the remark this answers is not in the document]
Comment (akornich) [reply Demos/macosAppSwift/macosAppSwift/ContentView.swift]: corrected

File: Demos/macosAppSwift/macosAppSwift/ContentView.swift [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    }
+    
+    func createGuard() -> RollbarExceptionGuard {
+        
+        let config = RollbarConfig();
+        config.destination.accessToken = "2ffc7997ed864dda94f63e7b7daae0f3";
```
[the remark this answers is not in the document]
Comment (akornich) [reply Demos/macosAppSwift/macosAppSwift/ContentView.swift:92]: agree in general. but in this specific case - it is a demo app assumed to be experimented with by the SDK customers locally using pre-allocated Rollbar Project dedicated to the SDK testing and demo usage.

File: RollbarSwift/Sources/RollbarSwift/RollbarExceptionGuard.m (hunk 1 above)
Comment (akornich) [reply RollbarSwift/Sources/RollbarSwift/RollbarExceptionGuard.m]: just replied Xcode default indentation.

**Comment by akornich** [comment]
@bishopb , thank you!

**Pull request merged**